src/pages/Archive/Archive.js

```diff
@@ -0,0 +1,240 @@
+        result[parentGroupId] = true;
+      });
+    }
+  });
+
+  return result;
+};
+
+const buttonsCreator = (
+  { GROUP_INFO_URI_FACTORY, GROUP_DETAIL_URI_FACTORY },
+  setRootGroup
+) => groupId =>
+  <span>
+    <Button
+      bsStyle="default"
+      bsSize="xs"
+      onClick={ev => {
+        ev.stopPropagation();
+        setRootGroup(groupId);
+      }}
+    >
+      <SuccessIcon gapRight />
+      <FormattedMessage
+        id="app.group.setRoot"
+        defaultMessage="Set Root Group"
+      />
+    </Button>
+    <LinkContainer to={GROUP_INFO_URI_FACTORY(groupId)}>
+      <Button bsStyle="primary" bsSize="xs">
+        <GroupIcon gapRight />
+        <FormattedMessage id="app.group.info" defaultMessage="Group Info" />
+      </Button>
+    </LinkContainer>
+    <LinkContainer to={GROUP_DETAIL_URI_FACTORY(groupId)}>
+      <Button bsStyle="primary" bsSize="xs">
+        <GroupIcon gapRight />
+        <FormattedMessage id="app.group.detail" defaultMessage="Group Detail" />
+      </Button>
+    </LinkContainer>
+    <ArchiveGroupButtonContainer id={groupId} bsSize="xs" />
+  </span>;
+
+class Archive extends Component {
+  state = { showAll: false, search: '', rootGroup: null };
+
+  static loadAsync = (params, dispatch, { instanceId }) =>
+    Promise.all([
+      dispatch(fetchInstancesIfNeeded(instanceId)),
+      dispatch(fetchAllGroups())
+    ]);
+
+  componentWillMount() {
+    this.props.loadAsync(this.props.instanceId);
+  }
+
+  render() {
+    const { instance, groups, intl: { locale }, links } = this.props;
+
+    return (
+      <Page
+        resource={instance}
+        title={
+          <FormattedMessage id="app.archive.title" defaultMessage="Archive" />
+        }
+        description={
+          <FormattedMessage
+            id="app.archive.description"
+            defaultMessage="List of archived groups."
+          />
+        }
+        breadcrumbs={[
+          {
+            text: (
+              <FormattedMessage
+                id="app.archive.title"
+                defaultMessage="Archive"
+              />
+            ),
+            iconName: 'archive'
+          }
+        ]}
+      >
+        {data =>
+          <Box
+            title={
+              <FormattedMessage
+                id="app.archive.archivedGroups"
+                defaultMessage="Archived Groups"
+              />
+            }
+            unlimitedHeight
+          >
+            <React.Fragment>
+              <FilterArchiveGroupsForm
+                form="archive-filters"
+                initialValues={{ showAll: false, search: '' }}
+                onSubmit={data => {
+                  this.setState({
+                    showAll: Boolean(data.showAll),
+                    search: data.search || ''
+                  });
+                  return Promise.resolve();
+                }}
+              />
+              <Grid fluid>
+                <Row>
+                  <Col xs={12} style={{ padding: '0' }}>
```

Comment:
Create a `.no-padding` class in recodex.css (similar to `.no-margin) and use it.